spec.html

```diff
@@ -1037,6 +1037,28 @@ <h1>Runtime Semantics: StringIndexOf ( _string_, _searchValue_, _fromIndex_ )</h
           <p>This algorithm always returns -1 if _fromIndex_ &gt; the length of _string_.</p>
         </emu-note>
       </emu-clause>
+
+      <emu-clause id="sec-stringincludesat" aoid="StringIncludesAt">
+        <h1>Runtime Semantics: StringIncludesAt ( _string_, _target_, _atIndex_ )</h1>
```

Comment:
These parameter names make it more immediately obvious to me which is which.

```suggestion
        <h1>Runtime Semantics: StringIncludesAt ( _haystack_, _needle_, _atIndex_ )</h1>
```

Reply:
I came to appreciate haystack/needle naming ever since I saw it in PHP docs, however:
* it implies searching a _needle_ in a _haystack_, yet this op is rather matches a substring at given index;
* it may sound not formal enough for the spec?

Reply:
If you don't like the haystack/needle searching metaphor, there's plenty of other alternatives that are better than what is here right now: whole/part, string/substring, outer/inner, etc.

Reply:
Thank you, switched to string/substring @ 8a80eee.